src/screens/ProfileScreen/index.js

```diff
@@ -31,6 +31,7 @@ import CustomPressable from '../../components/CustomPressable';
 import FollowInfoRow from './elements/FollowInfoRow';
 import ImageCompressionUtils from '../../utils/image/compress';
 import LinkAndSocialMedia from './elements/LinkAndSocialMedia';
+import PostOptionModal from '../../components/Modal/PostOptionModal';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `PostOptionModal` is added, but it's not clear if this component is used in the file. If it's not used, it should be removed to keep the code clean and maintainable.
```diff
- import PostOptionModal from '../../components/Modal/PostOptionModal';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->